src/filesystem.ts

```diff
@@ -1,9 +1,9 @@
-import { Issue } from "./issue";
+import {GitlabIssue} from "./issue";
```

Comment:
Please can you type hint against the Issue interface rather than the GitlabIssue class. See comment below about adding "filename" to the interface.